core/src/main/scala/io/delta/tables/DeltaTable.scala

```diff
@@ -653,7 +653,13 @@ object DeltaTable {
   }
 
   /**
-   * Create a DeltaTable using the given table or view name using the given SparkSession.
+   * Instantiate a [[DeltaTable]] object using the given table or view name, If the given
+   * tableOrViewName is invalid (i.e. either no table exists or an existing table is not a
+   * Delta table), it throws a `not a Delta table` error.
+   *
+   * The given tableOrViewName can also be the absolute path of a delta datasource(i.e. delta.path),
```

Comment:
nit: `datasource (i.e. ____)` (add a space)
nit: delta.\`path\` instead of delta.path